### PR TITLE
Ading Coreflux MQTT Broker 

### PR DESCRIPTION
**Pull request by HugoAVaz**

### CODE REVIEW

File: _sites/software.md [hunk 1]
```diff
@@ -53,6 +53,13 @@ description: A collection of links to all important MQTT brokers/servers, MQTT c
          Cassandana is an open source MQTT message broker which is entirely written in Java. This project began its life as a fork of <a href="https://github.com/andsel/moquette">Moquette</a> , and later underwent some cleanup, optimization and adding extra features. Now it’s ready to work as an enterprise message broker.
          </div>
       </div>
+   <!-- Entry -->
+      <div class="panel-item">
+         <img src="{{ 'assets/img/coreflux.png' | relative_url }}" class="software-logo" alt="Coreflux Logo">
+         <div class="panel-item-description"><a href="https://www.coreflux.org"><h3>Coreflux</h3></a>
+          Coreflux is a a Data Hub, based in MQTT 3.11 and 5.0,  designed to handle vast amounts of data from various sources, be it IoT devices, databases, applications, or external systems. As data flows in, the Flux DSL is able to do processes and orchestrates workflows based on predefined FLux Scripts. The Asset Management system ensures that each device is correctly configured and operating optimally. Check the <a href="https://doc.coreflux.org">documentation</a>!
```
Comment (B-Weiss) [inline _sites/software.md:60]: There is one blank too many in the description. Would you mind correcting this?

**Review by B-Weiss — COMMENTED** [review]
Just a little typo

File: _sites/software.md (hunk 1 above)
Comment (HugoAVaz) [reply _sites/software.md:60]: You mean the description is too big? And the trail space in "   designed" ?

Comment (B-Weiss) [reply _sites/software.md]: Yes, the additional space before the word "designed" is what I was referring to. Thanks for fixing.

**Review by B-Weiss — APPROVED** [review]

**Pull request merged**